### ISSUE TITLE
`NoObjectForPrimaryHashId` in `fix1926.md`?

### ISSUE THREAD

**Issue by aryairani** [unisonweb/unison#2011]
I currently can't reproduce this, but I also haven't fixed it.

Input:
````
```ucm
.> builtins.merge
```

```unison
> 'sq

sq = 2934892384
```

```unison
> 'sq

sq = 2934892384
```
````

Output:
```
⚙️   Processing stanza 3 of 3.unison: NoObjectForPrimaryHashId (HashId 156)
CallStack (from HasCallStack):
  error, called at ./U/Codebase/Sqlite/Queries.hs:98:35 in unison-codebase-sqlite-0.0.0-Ggbkzu9knpGBvCtyJKRnTD:U.Codebase.Sqlite.Queries
  throwError, called at ./U/Codebase/Sqlite/Queries.hs:116:20 in unison-codebase-sqlite-0.0.0-Ggbkzu9knpGBvCtyJKRnTD:U.Codebase.Sqlite.Queries
  orError, called at ./U/Codebase/Sqlite/Queries.hs:247:39 in unison-codebase-sqlite-0.0.0-Ggbkzu9knpGBvCtyJKRnTD:U.Codebase.Sqlite.Queries
  expectObjectIdForPrimaryHashId, called at ./U/Codebase/Sqlite/Operations.hs:194:28 in unison-codebase-sqlite-0.0.0-Ggbkzu9knpGBvCtyJKRnTD:U.Codebase.Sqlite.Operations
  primaryHashToExistingObjectId, called at ./U/Codebase/Sqlite/Operations.hs:404:14 in unison-codebase-sqlite-0.0.0-Ggbkzu9knpGBvCtyJKRnTD:U.Codebase.Sqlite.Operations
  getCycleLen, called at src/Unison/Codebase/SqliteCodebase.hs:278:14 in unison-parser-typechecker-0.0.0-6U6boimwb8GAC5qrhLfs8h:Unison.Codebase.SqliteCodebase
  getCycleLen, called at src/Unison/Codebase/SqliteCodebase.hs:595:38 in unison-parser-typechecker-0.0.0-6U6boimwb8GAC5qrhLfs8h:Unison.Codebase.SqliteCodebase
fix1926.md EXCEPTION!!!: callCommand: /Users/arya/unison/.stack-work/install/x86_64-osx/f01948b09d5ee28a669faeb529a516a52d69624ee0a67a1726b5b3455c742232/8.10.4/bin/unison transcript unison-src/transcripts/fix1926.md (exit 1): failed
⚙  /Users/arya/unison/.stack-work/install/x86_64-osx/f01948b09d5ee28a669faeb529a516a52d69624ee0a67a1726b5b3455c742232/8.10.4/bin/unison transcript unison-src/transcripts/fix689.md
💥  fix1926.md
```

Here is the  https://gist.github.com/aryairani/a24553bf244f90882fb9e021689170ef

**Comment by ChrisPenner** [comment]
This seems somewhat similar to https://github.com/unisonweb/unison/issues/2122; I can reproduce that one on trunk, but can't reproduce this one.

The fact that it has `getCycleLen` in the error makes me hope it's fixed in `topic/rehash-codebase`; but without a working reproduction it's hard to know.

**Comment by ChrisPenner** [comment]
Presumably fixed by #2627, feel free to re-open if we're able to reproduce this again.
